setup.py

```diff
@@ -50,7 +50,7 @@ def read_file(file_name):
         exclude=["*.tests", "*.tests.*", "tests.*", "tests"],
     ),
     include_package_data=True,
-    python_requires=">=3.9, <3.10",
+    python_requires=">=3.9, <3.12",
```

Comment:
LGTM, just shouldn't we better remove 3.11 here as well, as the test for it fails?

Reply:
Ahh yes sorry, that should be corrected